src/stream/sink/webrtc_sink.rs

```diff
@@ -450,9 +449,25 @@ impl SinkInterface for WebRTCSinkInner {
             unreachable!()
         };
 
+        // Block data flow to prevent any data before set Playing, which would cause an error
+        let Some(tee_src_pad_data_blocker) = tee_src_pad
+            .add_probe(gst::PadProbeType::BLOCK_DOWNSTREAM, |_pad, _info| {
+                gst::PadProbeReturn::Ok
+            }) else {
+                let _ = tee_src_pad
+                    .parent_element()
+                    .map(|parent| parent.release_request_pad(tee_src_pad));
```

Comment:
for_each to avoid the underscore result ?

Reply:
Oh, this is not an iter-map, it's a result-map.

Reply:
The solution is to use `if let Some() = .. { parent.release... }`

Reply:
nah, underscore is fine.

Reply:
Too late